index.html

```diff
@@ -6,83 +6,8 @@
 	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/atom-one-dark.min.css">
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jsoneditor/5.19.0/jsoneditor.min.css" />	
-	<style id="common">
-		/* General styles */
```

Comment:
Not sure whether we should remove these internal styles from here. They could be useful incase AJAX fails. As it is we replace the contents of this <style> element upon successful file fetch, so there won't be any redundancy.

Reply:
Yeah. We need to double check if this breaks any of the features.